extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +43,129 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
3 17.2573 172.097 17.448C172.47 17.6387 172.76 17.89 172.968 18.202H173.046V17.37H174.385V23.649C174.385 24.3683 174.242 24.9793 173.956 25.482C173.679 25.9847 173.293 26.366 172.799 26.626C172.305 26.886 171.733 27.016 171.083 27.016ZM171.096 22.791C171.443 22.791 171.764 22.7087 172.058 22.544C172.353 22.3707 172.591 22.1237 172.773 21.803C172.955 21.4737 173.046 21.0793 173.046 20.62C173.046 20.1433 172.955 19.7447 172.773 19.424C172.591 19.0947 172.353 18.8477 172.058 18.683C171.764 18.5183 171.443 18.436 171.096 18.436C170.75 18.436 170.425 18.5227 170.121 18.696C169.827 18.8607 169.588 19.1033 169.406 19.424C169.224 19.7447 169.133 20.1433 169.133 20.62C169.133 21.088 169.224 21.4867 169.406 21.816C169.588 22.1367 169.827 22.3793 170.121 22.544C170.425 22.7087 170.75 22.791 171.096 22.791ZM176.191 24V14.692H177.595V24H176.191ZM182.366 24.208C181.716 24.208 181.135 24.0563 180.624 23.753C180.112 23.4497 179.709 23.0337 179.415 22.505C179.129 21.9763 178.986 21.374 178.986 20.698C178.986 20.0653 179.124 19.4803 179.402 18.943C179.679 18.4057 180.065 17.9767 180.559 17.656C181.061 17.3267 181.638 17.162 182.288 17.162C182.972 17.162 183.553 17.3093 184.03 17.604C184.515 17.8987 184.883 18.3017 185.135 18.813C185.386 19.3243 185.512 19.9007 185.512 20.542C185.512 20.6373 185.507 20.724 185.499 20.802C185.499 20.88 185.494 20.9407 185.486 20.984H180.364C180.433 21.6427 180.663 22.1367 181.053 22.466C181.451 22.7953 181.902 22.96 182.405 22.96C182.855 22.96 183.228 22.8603 183.523 22.661C183.817 22.453 184.051 22.1973 184.225 21.894L185.382 22.453C185.096 22.973 184.706 23.3977 184.212 23.727C183.718 24.0477 183.102 24.208 182.366 24.208ZM182.301 18.358C181.833 18.358 181.434 18.501 181.105 18.787C180.775 19.073 180.554 19.4543 180.442 19.931H184.121C184.103 19.7057 184.03 19.4717 183.9 19.229C183.77 18.9863 183.57 18.7827 183.302 18.618C183.042 18.4447 182.708 18.358 182.301 18.358Z" fill="#5E5E5E"/><circle cx="20" cy="20" r="16" fill="white"/><path d="M32.7667 20.3001C32.7667 19.4223 32.6889 18.5889 32.5556 17.7778H20V22.7889H27.1889C26.8667 24.4334 25.9222 25.8223 24.5222 26.7667V30.1001H28.8111C31.3222 27.7778 32.7667 24.3556 32.7667 20.3001Z" fill="#4285F4"/><path d="M20.0001 33.3333C23.6001 33.3333 26.6112 32.1333 28.8112 30.1L24.5223 26.7667C23.3223 27.5667 21.8001 28.0555 20.0001 28.0555C16.5223 28.0555 13.5779 25.7111 12.5223 22.5444H8.1001V25.9778C10.289 30.3333 14.789 33.3333 20.0001 33.3333Z" fill="#34A853"/><path d="M12.5223 22.5442C12.2445 21.7442 12.1001 20.8886 12.1001 19.9998C12.1001 19.1109 12.2556 18.2553 12.5223 17.4553V14.022H8.10008C7.18897 15.822 6.66675 17.8442 6.66675 19.9998C6.66675 22.1553 7.18897 24.1775 8.10008 25.9775L12.5223 22.5442Z" fill="#FBBC05"/><path d="M20.0001 11.9443C21.9668 11.9443 23.7223 12.6221 25.1112 13.9443L28.9112 10.1443C26.6112 7.98873 23.6001 6.6665 20.0001 6.6665C14.789 6.6665 10.289 9.6665 8.1001 14.0221L12.5223 17.4554C13.5779 14.2887 16.5223 11.9443 20.0001 11.9443Z" fill="#EA4335"/></svg>');
```

Comment:
@mszylkowski may have some best practices on how to optimize this svg

Reply:
This needs to follow the same structure as `i-amphtml-story-subscriptions-publisher-button`. The 'G' could be an png.

The text needs to be an i18n string.
<img width="265" alt="Screen Shot 2022-04-06 at 12 42 09 PM" src="https://user-images.githubusercontent.com/3860311/162035727-486e4f36-9a9e-4c8b-8811-e4f2e68d66b8.png">

Reply:
Holy moly! Unless there is a reason that it must be a svg I think we should make it dom and text.

Reply:
@jpettitt could you share the rationale behind why pure svg is used for SwG button? Or I am missing something here?

Reply:
Actually, the i18n swg buttons do exist [here](https://github.com/ampproject/amphtml/blob/main/third_party/subscriptions-project/swg-button.css#L115-L286). However, we cannot use them because our UI does not treat "Google" differently with rainbow colors. Instead, let's just make the "Subscribe with Google" pure string and do the i18n as we normally do.